builtins/amp-ad.js

```diff
@@ -284,6 +285,28 @@ export function installAd(win) {
       return loadPromise(this.iframe_);
     }
 
+    /** @override  */
+    unlayoutCallback() {
+      if (this.iframe_) {
+        removeElement(this.iframe_);
+        if (this.placeholder_) {
+          this.togglePlaceholder(true);
+        }
+        if (this.fallback_) {
+          this.toggleFallback(false);
+        }
+
+        this.iframe_ = null;
+        this.intersectionObserver_ = null;
+      }
+      return true;
+    }
+
+    /** @override  */
+    unlayoutOnPause() {
+      return true;
```

Comment:
I'm actually fine with either INACTIVE or PAUSE. My understanding from https://github.com/ampproject/amphtml/issues/2563 was we wanted PAUSE, though.